stytch/b2b/passwords/session/types.go

```diff
@@ -21,7 +22,10 @@ type ResetParams struct {
 	// SessionToken: A secret token for a given Stytch Session.
 	SessionToken string `json:"session_token,omitempty"`
 	// SessionJWT: The JSON Web Token (JWT) for a given Stytch Session.
-	SessionJWT string `json:"session_jwt,omitempty"`
+	SessionJWT             string             `json:"session_jwt,omitempty"`
+	SessionDurationMinutes int32              `json:"session_duration_minutes,omitempty"`
+	SessionCustomClaims    map[string]any     `json:"session_custom_claims,omitempty"`
+	Locale                 ResetRequestLocale `json:"locale,omitempty"`
```

Comment:
You'll want to update the API Docs first and re-run codegen against the new docs so all these new fields pick up field descriptions